cmd/snapd-generator/main.c

```diff
@@ -99,10 +99,16 @@ static bool file_exists(const char *path) {
 	return stat(path, &buf) == 0 && (buf.st_mode & S_IFMT) == S_IFREG;
 }
 
+// PATH may not be set (the case on 16.04), in which case this is the fallback
+// for looking up squashfuse / snapfuse executable.
+// Based on what systemd uses when compiled for systems with "unmerged /usr"
+// (see man systemd.exec).
+static const char *path_fallback = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin";
```

Comment:
I think we want ":/sbin:/bin" here as well (as 16.04 is unmerged). Feel free to foce push

Reply:
Nitpick, you can make this `static const char * const path_fallback` as well.

Reply:
Updated.